sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Emitter.cs

```diff
@@ -66,6 +66,7 @@ public static IHostBuilder ConfigureGeneratedFunctionMetadataProvider(this IHost
                                  {
                                      builder.ConfigureServices(s => 
                                      {
+                                         s.TryAddSingleton<IFunctionMetadataProviderAggregator, SkipDefaultFunctionMetadataProviderAggregator>();
```

Comment:
Would have to see whether `AddSingleton` or `TryAddSingleton` is better. For now using the `TryAdd` to potentially allow full customization. But I wonder if that is even possible due to the non-deterministic nature of the auto configuration extensions call order?